style.css

```diff
@@ -732,6 +783,29 @@
   margin-top: -92px;
 }
 
+#pop-up-window {
+  display: none;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 9999;
+  width: 100%;
+  height: 100%;
+  backdrop-filter: blur(10px);
+  flex-direction: column;
+  justify-content: center;
+}
+
+.pop-up-container {
+  align-self: center;
+  height: 90%;
+  width: 90%;
+  background-color: white;
+  padding: 3%;
+}
```

Comment:
- [ ] Ouch! this is close but there still seems to be some space for improvement as your design isn't in sync with the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=uWZZpVPwfoEXPnAJ-0) for the desktop popup. let me clarify below: 👇
     - Figma design:
       ![image](https://user-images.githubusercontent.com/104968236/231757248-b644c67b-8b0b-4859-aad7-050f8a5d5b5d.png)
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/231757660-f993a4e9-b667-42b2-830f-6c00aa4c9985.png)
     - [ ] 1st the width of the popup you're using should be increased to around 80-90% as shown in the Figma design. so please make sure to fix that. ✔
     - [ ] 2nd only the project description should be aligned to the left as shown in the Figma design. so please make sure to fix that. ✔
     - [ ]  3rd the list of technologies and the buttons should be aligned to the right as shown in the Figma design. so please make sure to fix that. ✔
     - [ ] 4th make sure you scale the image accordingly with the popup, now please notice the Fogma popup is at a 1440x1557 scale and your popup design might overflow when you have a full-width image but that's okay you can add a scroll bar to your popup so it doesn't overflow ([more info here](https://www.w3schools.com/css/css_overflow.asp)). ✔

Reply:
- [ ] I see you put a lot of effort into this and it's really impressive :heart_eyes: But you are almost there. 
On the mobile version, the project description and the technologies overflow the popup window. Kindly check it

_**HIT:** Add this property `overflow-y: scroll;` to this class `.pop-up-container` to make it scroll when the content is overflow_